catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -406,6 +427,13 @@ public static void setAutoCrashReportingEnabled(Context context, boolean isEnabl
 		}
 	}
 
+	public static void setCloudNotificationsEnabled(Context context, boolean isEnabled) {
+		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
+		editor.putBoolean(SETTINGS_CLOUD_NOTIFICATIONS, isEnabled);
```

Comment:
you can directly put `.commit()` at the end of this. Editor calls always return the Editor again ;)